ui/packages/catalog-ui-search/src/main/webapp/component/navigator/navigator.view.js

```diff
@@ -36,7 +37,122 @@ const visitFragment = fragment =>
 
 module.exports = plugin(
   Marionette.LayoutView.extend({
-    template: template,
+    template(props) {
+      return (
+        <React.Fragment>
+          <button
+            className="navigation-choice is-neutral choice-product is-button"
+            data-fragment="workspaces"
+          >
+            <span className="is-bold">{props.properties.branding}</span>
+            <span className="">{props.properties.product}</span>
+          </button>
+          <div className="is-divider" />
+          <div className="navigation-links">
+            <button
+              className="navigation-choice is-neutral choice-workspaces is-button"
+              data-fragment="workspaces"
+            >
+              <span className="fa fa-book" />
+              <span>Workspaces</span>
+              <div className="workspaces-indicator" />
+            </button>
+            <div className="workspaces-save" />
+            <button
+              className="navigation-choice is-neutral choice-upload is-button"
+              data-fragment="ingest"
+            >
+              <span className="fa fa-upload" />
+              <span>Upload</span>
+            </button>
+            <button
+              className="navigation-choice is-neutral choice-sources is-button"
+              data-fragment="sources"
+            >
+              <span className="fa fa-cloud" />
+              <IntlProvider locale="en">
+                <FormattedMessage
+                  id="sourcesKeyword"
+                  defaultMessage={`{sources}`}
+                  values={{ sources: properties.keywords['Sources'] }}
+                />
+              </IntlProvider>
+              <span className="sources-indicator fa fa-bolt" />
+            </button>
+            <button
+              className="navigation-choice is-neutral choice-search-forms is-button"
+              data-fragment="forms"
+            >
+              <span className="fa cf cf-search-forms" />
+              <span>Search Forms</span>
+              <div className="forms-indicator" />
+            </button>
+            <button
+              className="navigation-choice is-neutral choice-result-forms is-button"
+              data-fragment="resultForms"
+            >
+              <span className="fa cf cf-result-forms" />
+              <span>Result Forms</span>
+              <div className="forms-indicator" />
+            </button>
+            <div className="navigation-extensions" />
+          </div>
+          {props.recent ? (
```

Comment:
✏️ Consider doing `props.recent && <>..._some_html_...</>` if you don't really have a an else clause